docs/How-to-release.md

```diff
@@ -0,0 +1,204 @@
+account if a quality vote is called for this build.
+
+Release notes:
+
+ * https://github.com/apache/skywalking-cli/blob/v$VERSION/CHANGES.md
+
+Release Candidate:
+
+ * https://dist.apache.org/repos/dist/dev/skywalking/cli/$VERSION
+ * sha512 checksums
+   - sha512xxxxyyyzzz apache-skywalking-cli-bin-x.x.x.tgz
+   - sha512xxxxyyyzzz apache-skywalking-cli-src-x.x.x.tgz
+
+Release Tag :
+
+ * (Git Tag) v$VERSION
+
+Release Commit Hash :
+
+ * https://github.com/apache/skywalking-cli/tree/<Git Commit Hash>
+
+Keys to verify the Release Candidate :
+
+ * http://pgp.mit.edu:11371/pks/lookup?op=get&search=0x8BD99F552D9F33D7 corresponding to kezhenxu94@apache.org
+
+Guide to build the release from source :
+
+ * https://github.com/apache/skywalking-cli/blob/v$VERSION/docs/How-to-release.md
+
+A vote regarding the quality of this test build will be initiated
+within the next couple of days.
+```
+
+## Wait at least 48 hours for test responses
+
+Any PMC, committer or contributor can test features for releasing, and feedback.
+Based on that, PMC will decide whether to start a vote or not.
+
+## Call for vote in dev@ mailing list
+
+Call for vote in `dev@skywalking.apache.org`
+
+```text
+Subject: [VOTE] Release Apache SkyWalking CLI version $VERSION
+
+Content:
+
+Hi All,
+This is a call for vote to release Apache SkyWalking CLI version $VERSION.
+
+Release notes:
+
+ * https://github.com/apache/skywalking-cli/blob/v$VERSION/CHANGES.md
+
+Release Candidate:
+
+ * https://dist.apache.org/repos/dist/dev/skywalking/cli/$VERSION
+ * sha512 checksums
+   - sha512xxxxyyyzzz apache-skywalking-cli-src-x.x.x.tgz
+   - sha512xxxxyyyzzz apache-skywalking-cli-bin-x.x.x.tgz
+
+Release Tag :
+
+ * (Git Tag) v$VERSION
+
+Release Commit Hash :
+
+ * https://github.com/apache/skywalking-cli/tree/<Git Commit Hash>
+
+Keys to verify the Release Candidate :
+
+ * https://dist.apache.org/repos/dist/release/skywalking/KEYS
+
+Guide to build the release from source :
+
+ * https://github.com/apache/skywalking-cli/blob/v$VERSION/docs/How-to-release.md
+
+Voting will start now (xxxx date) and will remain open for at least 72 hours, all PMC members are required to give their votes.
+[ ] +1 Release this package.
+[ ] +0 No opinion.
+[ ] -1 Do not release this package because....
+```
+
+## Vote Check
+
+All PMC members and committers should check these before voting +1.
+
+1. Features test.
+1. All artifacts in staging repository are published with `.asc`, `.md5`, and `sha` files
+1. Source codes and distribution packages (`apache-skywalking-cli-{src,bin}-$VERSION.tgz`)
+are in `https://dist.apache.org/repos/dist/dev/skywalking/cli/$VERSION` with `.asc`, `.sha512`.
+1. `LICENSE` and `NOTICE` are in source codes and distribution package.
+1. Check `shasum -c apache-skywalking-cli-{src,bin}-$VERSION.tgz.sha512`
+1. Build distribution from source code package by following this [the build guide](#build-and-sign-the-source-code-package).
+1. Apache RAT check.
```

Comment:
I think you don't have rat check, yes?

Reply:
> I think you don't have rat check, yes?

Actually we have, docs are updated with the command